test/Common/HttpApiHandlerTests.cs

```diff
@@ -807,6 +807,42 @@ public async Task GetQueryWithoutRuntimeStatus_is_Success()
             Assert.Equal("Completed", actual[1].RuntimeStatus);
         }
 
+        [Fact]
+        [Trait("Category", PlatformSpecificHelpers.TestCategory)]
+        public async Task HandleGetStatusRequestAsync_Correctly_Parses_InstanceId_With_Spaces()
+        {
+            var instanceId = "test instance id with spaces";
+            var list = (IList<DurableOrchestrationStatus>)new List<DurableOrchestrationStatus>

```

Comment:
Can we update this line and remove `(IList<DurableOrchestrationStatus>)`?
```suggestion
            var list = new List<DurableOrchestrationStatus>

```